### ISSUE TITLE
[F glutil.cpp:338] eglInitialize() failed

### ISSUE THREAD

**Issue by tanvikes** [nv-tlabs/GET3D#129]
Hi Team,

I tried to run the code with the pre-trained models for Inference using the Google Colab reference notebook and by creating a docker image as well. However, I am facing the same issue in both the cases.

Creating output directory...
Launching processes...
Setting up PyTorch plugin "upfirdn2d_plugin"... Done.
Setting up PyTorch plugin "bias_act_plugin"... Done.
Setting up PyTorch plugin "filtered_lrelu_plugin"... Done.
==> resume from pretrained path /GET3D/get3d_release/shapenet_car.pt
==> generate
/GET3D/training/networks_get3d.py:430: UserWarning: torch.range is deprecated and will be removed in a future release because its behavior is inconsistent with Python's range builtin. Instead, use torch.arange, which produces values in [start, end).
  camera_theta = torch.range(0, n_camera - 1, device=self.device).unsqueeze(dim=-1) / n_camera * math.pi * 2.0
**[F glutil.cpp:338] eglInitialize() failed**
**Aborted**

Can you please  guide me on how to fix this issue?

**Comment by tanvikes** [comment]
Referred to an older ticket and it worked

**Comment by boytjj** [comment]
@tanvikes I got the same issue result as yours,
how you fix it?
reintall torch+cuda to an older version?
